modules/dop/services/autotest/global_config.go

```diff
@@ -307,7 +307,7 @@ func (svc *Service) DeleteGlobalConfig(req apistructs.AutoTestGlobalConfigDelete
 	if _, err := svc.cms.DeleteCmsNsConfigs(utils.WithInternalClientContext(context.Background()), &cmspb.CmsNsConfigsDeleteRequest{
 		Ns:             req.PipelineCmsNs,
 		PipelineSource: apistructs.PipelineSourceAutoTest.String(),
-		DeleteNs:       false,
+		DeleteNs:       true,
```

Comment:
how this bug introduced? just forget to set `true`?

Reply:
Parameter ‘deleteNS' is true in version release/1.1,but it was be set as false after pb refector